extensions/interactions/TextInput/TextInput.js

```diff
@@ -81,3 +81,21 @@ oppia.directive('oppiaShortResponseTextInput', [
     };
   }
 ]);
+
+oppia.factory('textInputRulesService', [function() {
+  return {
+    Equals: function(answer, inputs) {
+      return answer.toLowerCase() == inputs.x.toLowerCase();
+    },
+    CaseSensitiveEquals: function(answer, inputs) {
+      return answer == inputs.x;
+    },
+    StartsWith: function(answer, inputs) {
+      return answer.indexOf(inputs.x, 0) == 0;
```

Comment:
I think this should also not be case sensitive? It isn't case sensitive in the backend implementation.